database/neo4j/index.js

```diff
@@ -216,9 +205,60 @@ module.exports.getSortedRestaurantReviews = ({ restaurantId, list, sorting }) =>
         }
         // console.log(reviewsArr);
         return reviewsArr;
-      })
-      .catch((error) => {
-        console.log(error);
-      })
+      });
   }
 };
+
+module.exports.addOneReview = (newReview) => {
+  const session = driver.session();
+  let {
+    restaurantId,
+    reviewerId,
+    comments,
+    ambiance,
+    service,
+    noise,
+    overall,
+    recomment,
+    food,
+    date,
+    tag,
+    reviewId,
+  } = newReview;
+
+  return session
+    .run(`
+      MATCH (rest:Restaurant), (reviewer:Reviewer)
+      WHERE rest.restaurantId = $restaurantId AND reviewer.reviewerId = $reviewerId
+      CREATE (reviewer)-[:WRITTEN_BY {date: $date}]->(review:Review {
+        comments: $comments,
+        ambiance: $ambiance,
+        service: $service,
+        noise: $noise,
+        overall: $overall,
+        recomment: $recomment,
+        food: $food,
+        reviewId: $reviewId
+      })-[:TAGGED_AS {tag: $tag}]->(rest)
+      RETURN review
+      `,
+      {
+        restaurantId: restaurantId.toString(),
+        date: date,
+        reviewerId: reviewerId,
+        comments: comments,
+        ambiance: ambiance,
+        service: service,
+        noise: noise,
+        overall: overall,
+        recomment: recomment,
+        food: food,
+        date: date,
+        tag: tag,
+        reviewId: reviewId
+      })
```

Comment:
I'm not certain of this off the top of my head, but couldn't you reduce this even further with templating? I.e. 
{ 
  date,
  reviewerId,
  etc,...
}

Otherwise looks good to me!